File: csharp/console/oops/class1/Backup/class1/Class1.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace class1
{
    class Class1
    {
        public static void Main()
        { 
            student s1=new student(1,"Ravi");
            s1.disp();


            student[] s=new student[10];

            s[0]=new student(1,"Sunil");
            s[0].disp();

            s[1] = new student(2, "Ravi");
            s[1].disp();
            
        }

    }
    class student
    {
        public int rno;
        public string name;
        public student(int a, string n)
        {
            rno = a;
            name = n;
        }
        public void disp()
        {
            Console.WriteLine("Rno=" + rno);
            Console.WriteLine("name=" + name);
        }
    }
}

File: VB.NET/console/oops/inherts/inherts/Class1.vb
Module Class1

    Sub Main()
        Dim s As New detail()
        s.getname(1, "Sunil")
        s.disp()



    End Sub

End Module
Public Class customer
    Friend cno As Integer
    Public name As String

    Public Sub getname(ByVal id As Integer, ByVal n As String)
        cno = id
        name = n
    End Sub


End Class
Class detail
    Inherits customer
    Public pname As String

    Public Sub disp()
        pname = "PayRoll"
        Console.WriteLine("Derived name=" + pname)

        Console.WriteLine("rno=" + cno.ToString)
        Console.WriteLine("name=" + name)
    End Sub



End Class
